tensorboardX/record_writer.py

```diff
@@ -66,7 +66,10 @@ def __del__(self):
         self.close()
 
     def bucket_and_path(self):
-        bp = self.path.split("/")
+        path = self.path
+        if path.startswith("s3://"):
+                path = path[len("s3://"):]
```

Comment:
Sorry - bad spacing here. Can you just indent by 4 spaces? Thanks.

Reply:
https://github.com/lanpa/tensorboardX/commit/bdcc1f83235bb5d8554edbfb298550dd00214a5b
Changing that will break setup.py again.

Reply:
dammit. Well, I'm not sure the correct way around, but on my mac, I am not able to `pip install git+https://github.com/lanpa/tensorboardX`. Maybe this might work? I'm not super familiar with Python build systems: https://stackoverflow.com/questions/20194565/running-custom-setuptools-build-during-install/20196065#20196065